api/v2beta2/groupversion_info.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2020 The Flux authors
```

Comment:
Across the new files:
```suggestion
Copyright 2022 The Flux authors
```